santoni.json

```diff
@@ -0,0 +1,28 @@
+{
+  "response": [
+    {
+        "maintainer": "Anar Jabiyev (Jabiyeff)",
+        "oem": "Redmi",
+        "device": "Redmi 4X",
+        "filename": "crDroidAndroid-12.0-20220116-santoni-v8.1.zip",
+        "download": "https://sourceforge.net/projects/crdroid/files/santoni/8.x/crDroidAndroid-12.0-20220116-santoni-v8.1.zip/download",
+        "timestamp": 1642362905,
+        "md5": "7ad361940f4e28d708d3afd3764a12cb",
+        "sha256": "f39276685520ac72ee9694cca2d7be711469a210e237bf198ea8ca60c4e15eaa",
+        "size": 719258572,
+        "version": "8.1",
+        "buildtype": "Beta",
```

Comment:
beta for now
https://github.com/crdroidandroid/android_device_xiaomi_santoni/blob/12.0/rootdir/fstab.qcom#L10#L11